ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -255,7 +259,12 @@ public static void setUpClass() throws Exception {
         AGG_SCHEMA
     );
 
-    final String testDir = Paths.get(TestUtils.tempDirectory().getAbsolutePath(), "client_integ_test").toString();
+    makeKsqlRequest(
```

Comment:
nit: can we move this up a few lines to be alongside the other `makeKsqlRequest()`?